scratch/benchmarks/fgpu_calibration/calibrate-n1.txt

```diff
@@ -0,0 +1,41 @@
+8000000000.0 100 100
```

Comment:
Are these distribution functions independent of all the various parameters except `n`?

Reply:
They're almost certainly not. What I'm hoping is that the slope is mostly independent while the offset is dependent. In other words, changing parameters will change the achievable throughput, but not the uncertainty. Unfortunately I don't think it's feasible (in time spent running calibrations) to validate that assumption.